CivicActions/FE-knowledgebase
# Strapi CMS

**QA Steps**

1. Clone this branch `git checkout -b my-local-branch origin/strapi-cms-ii`
2. Run `npm install`
3. Run `gatsby develop`
4. The local site should be visible on http://localhost:8000/
5. Go to http://localhost:8000/recipes to see the test recipe from the strapi cms.

**Heroku app**
1. Login to the app on: https://fe-knowledgebase-cms.herokuapp.com/admin
2. Click the Recipes link (in the sidebar)  then click the "Add new Recipes" button on the top right to add a new recipe.
3. Save and confirm that the recipe is visible in the api:  https://fe-knowledgebase-cms.herokuapp.com/recipes
4. Back in the local gatsby instance, restart the app by running `ctrl+c` to stop and then `gatsby develop` again.
5. In http://localhost:8000/recipes, confirm that the new recipe you added is visible.
